packages/backend/src/database/migrations/20211118105119_add_markdown_and_loom_dashboard_tiles.ts

```diff
@@ -0,0 +1,70 @@
+import { Knex } from 'knex';
```

Comment:
This looks great!

The only consideration I had was whether to put the loom and/or markdowns in their own tables. This would be useful, for example, if we wanted a library of all looms. Or to use the same loom in multiple places.

But in this case I think it makes perfect sense that the markdown is only defined on the dashboard. And in a way, Loom provides the "library" of looms.

All to say, this looks great